storage/retry_conformance_test.go

```diff
@@ -586,6 +587,9 @@ func TestRetryConformance(t *testing.T) {
 	if host == "" {
 		t.Skip("This test must use the testbench emulator; set STORAGE_EMULATOR_HOST to run.")
 	}
+	if runtime.GOOS == "darwin" {
```

Comment:
Let's remove this from the PR; seems non-relevant to this particular issue.

Reply:
👍 there was handling for this in the test script so I moved it in here, good to remove though.